tests/test_western/test_pedal_line.py

```diff
@@ -0,0 +1,39 @@
+from ..helpers import AppTest
+import pytest
+
+from neoscore.western.staff import Staff
+from neoscore.western.pedal_line import PedalLine
+from neoscore.core.units import ZERO, Mm
+from neoscore.core.point import Point
+
+class TestBarLine(AppTest):
+    def setUp(self):
+        super().setUp()
+        self.staff = Staff((Mm(0), Mm(0)), None, Mm(100))
```

Comment:
you can use `ORIGIN` instead of `(Mm(0), Mm(0)) here and in the several other places in this test that use it. (you just need to import it from `neoscore.core.point` of course)